File: src/VacancyManager/VacancyManager/Models/Comment.cs
using System;

namespace VacancyManager.Models
{
    public class Comment
    {
        public int CommentID { get; set; }
        public int UserID { get; set; }
        public int ConsiderationID { get; set; }
        public DateTime CreationDate { get; set; }

        public string Body { get; set; }

        public virtual User User { get; set; }
        public virtual Consideration Consideration { get; set; }
    }
}

File: src/VacancyManager/VacancyManager/Scripts/VM/Admin/model/Comment.js
Ext.define('VM.model.Comment', {
    extend: 'Ext.data.Model',
    idProperty: 'CommentID',
    fields:
    [
      { name: 'CommentID', type: 'int', useNull: true },
      { name: 'CreationDate', type: 'date', dateFormat: 'd.m.Y' },
      { name: 'Body', type: 'string' },
      { name: 'ApplicantId', type: 'int', useNull: true }, 
      { name: 'UserID', type: 'int' },
      { name: 'UserRoles', type: 'string' },
      { name: 'CommentatorName', type: 'string' },
      { name: 'Vacancy', type: 'string' },
      { name: 'ConsiderationID', type: 'int', useNull: true }

    ]
});